woeip/apps/air_quality/models.py

```diff
@@ -54,19 +54,20 @@ def __str__(self):
 
 
 class SessionData(TimeStampedModel):
-    """The raw data file generated during a session. Assumes one and only one file per sensor,
-    although multiple sensors can be linked to one session"""
+    """The raw data files generated during a session. Assumes one general and one gps file.
+    Multiple sensors can be linked to one session"""
     upload = models.FileField(upload_to='session_data')
+    upload_gps = models.FileField(upload_to='session_gps_data', default="")
```

Comment:
Suggestion for improving variable naming clarity: `sensor_file` and `gps_file` instead of `upload` and `upload_gps`.

Reply:
I'll add it to the list of redundant migrations :stuck_out_tongue: